src/draw_buffer.rs

```diff
@@ -27,21 +28,43 @@ impl Default for ColorAttachmentType {
 #[derive(Debug)]
 pub enum ColorAttachment {
     Renderbuffer(GLuint),
-    Texture(GLuint),
+    Texture(GLuint, GLuint),
```

Comment:
Can you make this `ColorAttachment::Texture { active: GLuint, complete: GLuint }`?